src/CareTogether.Core/Resources/Accounts/AccountsResource.cs

```diff
@@ -45,6 +45,21 @@ IEventLog<PersonAccessEvent> personAccessEventLog
             randomNumberGenerator = RandomNumberGenerator.Create();
         }
 
+        public async Task<Guid[]> GetValidOrganizationsAsync()
+        {
+            //WARNING: The read/write logic in this service needs to be designed carefully to avoid deadlocks.
```

Comment:
Is this warning relevant for this method? I kept it because I copied this method from `TryGetUserAccountAsync`, but not sure about the warning

Reply:
It's just generally there for awareness with those reader/writer locks; when we rewrite this I'd like to clean all that up.